### PR TITLE
Pkgbuild: spilt up into release cef, release desura and git pkgbuild

### PR DESCRIPTION
**Pull request by matthiaskrgr**

### CODE REVIEW

File: distro/archlinux/release/cef/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  cd build_cef
+  msg "Running cmake..."
+
+  CMAKE_ARGS="
+    -D32BIT_SUPPORT=$MULTILIB
+    -DCMAKE_INSTALL_PREFIX=/usr
+    -DBINDIR=/usr/bin
+    -DDATADIR=/usr/share/
+    -DRUNTIME_LIBDIR=/usr/lib/
+    -DBUILD_ONLY_CEF=ON"
```
Comment (matthiaskrgr) [inline distro/archlinux/release/cef/PKGBUILD:61]: What I found strange was that although I had this...

File: distro/archlinux/release/cef/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+package() {
+  cd "$srcdir/Desurium/build_cef"
+
+  msg "Compiling into pkgdir this time..."
+  make DESTDIR="$pkgdir/" install
+
+  msg "Compiling done"
+  
+  rm ${pkgdir}/usr/lib/desura/libwx_gtk2u_desura-2.9.so.3 # ugly hack to avoid file conflicts
```
Comment (matthiaskrgr) [inline distro/archlinux/release/cef/PKGBUILD:80]: .. I still had to do this to avoid file conflicts. Any ideas Karol?

Comment (matthiaskrgr) [inline distro/archlinux/release/cef/PKGBUILD:80]: @karolherbst

**Comment by matthiaskrgr** [comment]
I'll retry to use the official git release-download instead of git clone + checkout tag.
So please don't merge yet.

I'd also like to hear @lodle s opinion on this patch.

**Comment by matthiaskrgr** [comment]
Also, do we need cef at buildtime?

**Comment by karolherbst** [comment]
not for launching, but for using the internal web browser. In addition to that: how it is possible to install a specific version like 0.8.0_rc2?

**Comment by matthiaskrgr** [comment]
I made it so that we simply check out the package version

git checkout $pkgver

so as long as you we have a git tag representing this, you can change the pkgver= as you desire.

**Pull request merged**